packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -241,7 +242,7 @@ function handler(app: Application) {
       (label: Label) =>
         label.name === MERGE_ON_GREEN_LABEL ||
         label.name === MERGE_ON_GREEN_LABEL_SECURE
-    );
+    )?.name;
```

Comment:
Alternatively, you're ensuring label is present on the next line. You could confidently use `label.name` after that check.